src/constants.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  generateRelationshipType,
+  RelationshipClass,
+  StepEntityMetadata,
+  StepRelationshipMetadata,
+} from '@jupiterone/integration-sdk-core';
+
+export const DEFAULT_SCANNED_SINCE_DAYS = 7;
+export const DEFAULT_FINDINGS_SINCE_DAYS = 7;
+
```

Comment:
This doesn't super matter, but it might be nice to create some constants for step IDs that can be referenced across different steps:

```suggestion

export const steps  = {
  ACCOUNT: 'fetch-account',
  GROUPS: 'fetch-groups',
  ENDPOINTS: 'fetch-endpoints',
  CONFIGURATIONS: 'fetch-configurations',
  SCAN_FINDINGS: 'fetch-scan-findings',
}
```

Reply:
That's actually really nice! Prevents problems if strings aren't 100% identical. Added.